hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/UnhealthySubscriptionsGetProperties.java

```diff
@@ -0,0 +1,25 @@
+package pl.allegro.tech.hermes.management.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("unhealthy-subscription-get")
```

Comment:
There is a class with properties related to this mechanism: `pl.allegro.tech.hermes.management.config.HealthProperties`. Unfortunately its name is a little bit confusing - somebody may think that it is an application’s health check configuration. I suggest to change current structure to something like this:
```yaml
subscription-health-check: 
    maxLagInSeconds: 600
    maxOtherErrorsRatio: 0.5
    ...
    timeoutMillis: 8000
    threadPoolSize: 16
```

Reply:
I've changed its name to `SubscriptionHealthProperties ` and added timoeut and threadPool properties there